README.md

```diff
@@ -1,56 +1,26 @@
-<!---  Submission Readme Instructions  
-  Remove the warning below.
---->
-![warning](./challenges/challenge1/usecase/images/misc/warning.png) Before submitting your project repo, modify this README as instructed to meet the expectations of the judges.
-
-
-<!---  Submission Readme Instructions  
-  Welcome to the FINOS GAAD Hackathon!
-
-  This GitHub Repo represents a template for your project. It represents the central repository of all artifacts produced by your team. This repository will be referenced during the judging activity and after the event has completed.
-
-  Please follow the Template Instructions herein to update this README.
-
-  Remember to:
-    a) Remove all Template Instructions once complete.
-    b) Add the mandatory GitHub Topics.
---->
- 
-# Project Name
-<!--- Template Instructions  
-  Each Participating Team will have a unique name. Each Team will create a unique name for their project. Replace above "Project Name" with Participating Team Project Name which is different from the Team Name.
---->
+# API server modernization and Style Dictionary integration
 
 ## Project Details
-<!--- Template Instructions  
-  Provide your team specific details:
-
-    Challenge should have a numeric value {1, 2, etc}.
-
-    Event Track should have one of the following values: "Hack the Code", "Design Thinking", "Not Applicable".
---->
 
 | Event Attribute| Description |
 | --- | --- |
-| Project Team Name | xxxxxxxx |
-| Project Name | xxxxxxxx |
-| Challenge | xxxxxxxx |
-| Event Track | xxxxxxxx |
+| Project Team Name | The Accessibles |
+| Project Name | API server modernization and Style Dictionary integration |
+| Challenge | 1 |
+| Event Track | Hack the Code |
 
 ## Use Case Abstract
-<!--- Template Instructions  
-  Provide a brief description of the use case tackled by the team.
---->
+
+### Theme Builder API server modernization
+
+After reading through the Theme Builder source code our team identified an opportunity to improve the backend server by changing the underlying technology being used to build it. [Fastify](https://www.fastify.io/) is a modern open-source Node.js web server framework with first class Typescript support and a vibrant community.
+### Style Dictionary integration
+
+After looking at the JSON output generated by the Theme Builder our team identified and opportunity to use it as an input to another tool that automates the creation of stylesheets for multiple platforms not only the Web. The [Style Dictionary](https://amzn.github.io/style-dictionary/#/) is an open-source project that allows software teams to define styles once, in a way for any platform or language to consume.
```

Comment:
```suggestion
After looking at the currently available output formats, JSON and CSS, our team identified an opportunity to rapidly expand the available formats of the generated design tokens. By leveraging the team's previous experience building design systems, we decided to use the open source tool [Style Dictionary](https://amzn.github.io/style-dictionary/#/) to generate design tokens that can be consumed by many platforms, including Android, iOS, web, JS, and more. This could have a massive impact on the adoption of the theme builder by generating design tokens for many platforms.
```